src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+$secondary: #616070;
+$btn-white-color: #fff;
+$btn-turquoise-color: #00acdc;
+$main-acent: #060b35;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-acent;
+    margin-bottom: 4px;
+    margin-top: 0;
+  }
+
+  &code {
+    line-height: 14px;
+    text-align: left;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &price-value {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &price-header {
+    font-size: 12px;
+    text-align: left;
+    color: #616070;
+  }
```

Comment:
The &price-header should be &__price-header to follow BEM naming conventions, which state that elements of a block should be named using a double underscore.